cmd/immudb/command/root.go

```diff
@@ -59,24 +59,35 @@ Setting the logging level and other options through environment variables:
 func (cl *Commandline) Immudb(immudbServer server.ImmuServerIf) func(*cobra.Command, []string) error {
 	return func(cmd *cobra.Command, args []string) (err error) {
 		var options *server.Options
+
 		if options, err = parseOptions(); err != nil {
 			return err
 		}
+
 		immudbServer := immudbServer.WithOptions(options)
-		if options.Logfile != "" {
-			if flogger, file, err := logger.NewFileLogger("immudb ", options.Logfile); err == nil {
-				defer file.Close()
-				immudbServer.WithLogger(flogger)
-			} else {
-				c.QuitToStdErr(err)
-			}
+
+		// initialize logger for immudb
+		flogger, file, err := logger.NewLogger(&logger.Options{
```

Comment:
problably the `logger` should be closeable and the file (if any) is closed when calling `Close` method.
```
logger, err := logger.NewLogger(...)
if err != nil {
  ...
}
defer logger.Close() // file is closed under the hood, if any
```

Reply:
Yes, I was thinking of adding it to the interface, this would help in avoiding this check of getting the file back and closing it

Reply:
added the Close method to Logger interface